hack/stern/run-stern.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+
+# Determine where this script is and make it the cwd
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" &> /dev/null && pwd)"
+cd "$SCRIPT_DIR"
+
+# Default variables for the script
+STOP="false"
+LOGFILE=""
+
+# Help
+helpmsg() {
+cat <<HELPMSG
+This script controls stern logging binary for istio (upstream) and service mesh (downstream) containers + demo apps .
+    Valid command line arguments:
+    -l|--logfile <filename.json>: Required logfile name (default: Cypress uses current .feature file name, i.e. 'apps.feature.json')
+    -s|--stop <true|false>: If true, will stop any instances of the stern spawned by this script (default: false).
+    -h|--help : this message
+HELPMSG
+}
+
+# process command line args
+while [[ $# -gt 0 ]]; do
+  key="$1"
+  case $key in
+    -l|--logfile)
+      LOGFILE="$2"
+      shift;shift
+      ;;
+    -s|--stop)
+      STOP="$2"
+      shift;shift
+      ;;
+    -h|--help)
+      helpmsg
+      exit 1
+      ;;
+    *)
+      echo "Unknown argument [$key]. Aborting."
+      exit 1
+      ;;
+  esac
+done
+
+run_stern() {
+  local resource="$1"
+  local namespace="$2"
+  bash -c "exec -a sternLogs \"$SCRIPT_DIR/stern\" $resource $namespace --tail 0 -o json --only-log-lines --max-log-requests 150 >> $LOGFILE 2>&1" &
```

Comment:
I'm still -1 on outputting json logs. It makes the logs harder to read without any benefit right now.
```suggestion
  bash -c "exec -a sternLogs \"$SCRIPT_DIR/stern\" $resource $namespace --tail 0 --only-log-lines --max-log-requests 150 >> $LOGFILE 2>&1" &
```

Reply:
As Mazz mentioned, feeding ETL pipeline with unstructured data is pain. The goal for now is to get good baseline data in machine readable format - so the benefit is we can store and parse the logs in unified format. 

Let's create issue about this flag if you are concerned about readability when using stern logs locally.